packages/edit-site/src/components/style-book/types.ts

```diff
@@ -0,0 +1,27 @@
+type Block = {
+	name: string;
+	attributes: Record< string, unknown >;
+	innerBlocks: Block[];
+};
+
+export type StyleBookCategory = {
+	title: string;
+	slug: string;
+	blocks?: string[];
+	exclude?: string[];
+	subcategories?: StyleBookCategory[];
+};
+
+export type BlockExample = {
+	name: string;
+	title: string;
+	category: string;
+	blocks: Block;
```

Comment:
Should this be the single Block, an array of blocks e.g. `blocks: Block[];`, or either a single Block or array of Blocks: `blocks: Block | Block[];`?

`getExamples` looks to assign a single block for non-heading blocks, while the example for the heading block is an array of blocks.

The naming of the property indicates an array to me.

FWIW the [draft PR](https://github.com/WordPress/gutenberg/pull/65692) to add a color tab to the Style Book with arbitrary palette color examples [wraps the color blocks in a Grid block](https://github.com/WordPress/gutenberg/pull/65692/files#diff-1b402738fac82aad4bdb2b462dc8e63b59fe80957a7b876d043d9ef110b59a57R82-R96) to provide layout. So it could work with just a single Block.

Reply:
> Block | Block[];

Oh yes, this! 

Not sure what a better name would be, but I'll fix the type value and mull over it.